common.php

```diff
@@ -115,7 +138,7 @@ function display_navbar() {
 	<header class="container-fluid py-3">
 		<div class="row">
 			<div class="col-sm-4">
-				<a href="../"><img class="KU_image" src="../images/eecs_logo.png" height="60"></a>
+				<a href="../"><img class="KU_image" src="/images/eecs_logo.png" height="60"></a>
```

Comment:
I've been avoiding absolute paths since they won't work on my setup, but I figured that would have to change at some point